### PR TITLE
Add new flex options for iNET

### PR DESCRIPTION
**Pull request by redekok**
We’ve added:
- Nodes and edges
- Input statements
- Queries for relevant charts and tables

https://github.com/quintel/etsource/commit/fed645868689874f383b8b002469511766883ffc -> We have changed the location of the `energy_flexibility_pumped_storage_electricity`. First, it was connected to the HV network, now it is connected to the excess electricity node like all other flex options. I think it doesn’t matter for the performance of the flex option, but connecting the node to excess electricity is how it should be, right @antw ? (see also this comment https://github.com/quintel/etsource/issues/1744#issuecomment-428504470)

### CODE REVIEW

**Comment by antw** [comment]
> but connecting the node to excess electricity is how it should be, right @antw ?

I'm not an expert on this. It seems like it should be connected be connected to "excess" with everything else, but as I don't know why it was originally attached to the HV network I cannot say for sure.

File: nodes/households/households_flexibility_p2p_electricity.converter.ad [hunk 1]
```diff
@@ -3,7 +3,7 @@
 - output.electricity = 0.85
 - output.loss = elastic
 - groups = [cost_flexibility, lv_net_flexibility, p2p]
-- merit_order.group = power_to_power
+- merit_order.group = household_batteries
```
Comment (antw) [inline nodes/households/households_flexibility_p2p_electricity.converter.ad]: I'll need to write a migration in ETEngine to update existing scenarios to use this new name.

Comment (marliekeverweij) [reply nodes/households/households_flexibility_p2p_electricity.converter.ad]: Okay, good point! Forgot about that.

**Comment by marliekeverweij** [comment]
> I don't know why it was originally attached to the HV network 

I think it was connected to the HV accidentally...

**Pull request merged**